website/docs/cloud-docs/stacks/configure.mdx

```diff
@@ -27,8 +27,12 @@ You can configure a Stack by performing the following steps:
 1. Select the Stack you want to configure.  
 1. Click **Stack settings** in the navigation menu.  
 1. Update any of the following settings:  
-   * Name   
-   * Description  
+   * Name
+   * Description
+   * Project
+        * To move a stack, you must have the "Manage all Projects" organization permission or explicit team admin privileges on both the source and destination projects.
+        * Moving a Stack may alter or remove access for some teams based on their [project-level permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions).
+        * Upstream Stacks must be in the same project as their downstream Stacks. If you move a Stack to another project, the inputs that depend on it will break. Move upstream and downstream Stacks together to avoid disruptions. Learn more about [passing data from one Stack to another](/terraform/language/stacks/deploy/pass-data).
```

Comment:
I wasn't sure if bullet points were the best way to display this information - open to an alternative format!